### ISSUE TITLE
DeferredContent: AccessibilityDoc is missing a closing quotation mark in `data`

### ISSUE THREAD

**Issue by neilmerton** [primefaces/primevue#3719]
### Describe the bug

<img width="1142" alt="2023-03-09 at 18 37" src="https://user-images.githubusercontent.com/505027/224123416-e25176f5-abb5-4e57-984e-5651a6b6e829.png">

There's a quotation mark missing, I'll create a PR to fix this.

### Reproducer

https://primevue.org/deferredcontent/#accessibility

### PrimeVue version

3.24.0

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vue CLI App
